dev/com.ibm.ws.jsf.2.3_fat/publish/servers/jsf23CDIGeneralServer/server.xml

```diff
@@ -1,5 +1,5 @@
 <!--
-    Copyright (c) 2017, 2018 IBM Corporation and others.
+    Copyright (c) 2017, 2022 IBM Corporation and others.
```

Comment:
This server.xml has logging included, too

Reply:
Good catch, fixed!